src/olympia/reviewers/views.py

```diff
@@ -1517,3 +1520,12 @@ def retrieve(self, request, *args, **kwargs):
             instance=version, data={'parent_version': objs['parent_version']}
         )
         return Response(serializer.data)
+
+
+@bigquery_api_view(json_default=dict)
+@any_reviewer_required
+@reviewer_addon_view_factory
+@non_atomic_requests
+def usage_per_version(request, addon):
```

Comment:
So this is not an API, it uses regular session auth. Is that going to work for us, given that our session auth cookie is sent with `httpOnly` ?

Reply:
Mmmm yeah that should be fine actually - we shouldn't be trying to _access_ the cookie, and we're on the same domain.